.golangci.yml

```diff
@@ -80,6 +80,7 @@ issues:
       linters:
         - mnd
         - goconst
+        - dupl
```

Comment:
can you move this to a different commit and put a few words about what it checks?

Reply:
yeap will do. I didn't want to add complexity in the tests by making constants and extra functions, therefore I decided to ignore dupl in tests in favour of readability

Reply:
done, PTAL